eth2/simulation_args/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "simulation_args"
+version = "0.1.0"
+authors = ["Greg Trowbridge <gjtrowbridge@gmail.com>"]
```

Comment:
Not sure if you want your personal email or your consensys one here.

Reply:
I actually didn't even notice this, so thanks for the call-out!

For now I think my personal email makes more sense, since that's what's attached to my github account, and because my consensys email will go away after today.

That said, I also won't mind at all if this gets changed to anything else in the future.